packages/jest-core/src/runGlobalHook.ts

```diff
@@ -60,6 +60,7 @@ export default async ({
           transformer.transformSource(filename, code, false).code || code,
         {
           exts: [extname(modulePath)],
+          ignoreNodeModules: false,
```

Comment:
I think if we do this, we need to make sure it respects the `transformIgnorePatterns`

Reply:
It does that already